msg_helpers_test.go

```diff
@@ -209,6 +223,11 @@ func TestPackDataAplPrefix(t *testing.T) {
 			if !bytes.Equal(tt.expect, out[:off]) {
 				t.Fatalf("expected output %02x, got %02x", tt.expect, out[:off])
 			}
+			// Make sure the packed bytes would be accepted by its own unpack
+			_, _, err = unpackDataAplPrefix(out, 0)
```

Comment:
not absolutely necessary for the purpose of this PR but I wanted to do double-check the fix this way.